src/main/webapp/views/common/registration.html

```diff
@@ -37,7 +37,7 @@ <h5 class="modal-title" id="exampleModalLongTitle">Account Information</h5>
               <input type="tel" name="phone_number" id="phone_number" class="form-control" placeholder="8112345678" required>
             </div>
              <div class="form-group">
-              <label for="altText">Alt text for profile picture: </label>
+              <label for="altText">Profile picture description: </label>
               <input type="text" name="altText" id="altText" class="form-control" required>
```

Comment:
Nit: Maybe put in the placeholder something to explain why the description is needed? For example: `Please provide a meaningful description for screen reader users...`